.github/workflows/sync_theme.yml

```diff
@@ -7,7 +7,7 @@ on:
   schedule:
      - cron: "00 14 * * *"
 
-  workflow_dispatch:
+  workflow_dispatch: null
```

Comment:
what does this mean? `workflow_dispatch` allows an admin to run the action manually. Does this turn that feature off? It's a nice feature to enable.

Reply:
It is virtually the same but being explicit instead instead of leaving it blank. We should still be able to trigger workflows manually using this syntax.